eng/pipelines/vmr-sync-internal.yml

```diff
@@ -9,6 +9,7 @@ trigger:
     - internal/release/*.0.2xx
     - internal/release/*.0.3xx
     - internal/release/*.0.4xx
+    - internal/release/8.0.4*
```

Comment:
Why isn't the previous branch filter `internal/release/*.0.4xx` not working here?

Reply:
Oh, I see it's because it's 402, not 4xx.

Reply:
Going forward, we should be using * instead of xx because any of the feature bands could have an OOB release like 402.

Reply:
I wonder if AzDO can handle two `*`?

Reply:
It seems like the branching for OOB releases always finds ways to break these triggers :)